pkgs/sagemath-repl/known-test-failures.json

```diff
@@ -0,0 +1,165 @@
+{
+    "sage.doctest.control": {
+        "failed": true,
+        "ntests": 0
+    },
+    "sage.doctest.external": {
+        "ntests": 41
+    },
+    "sage.doctest.fixtures": {
+        "failed": true,
+        "ntests": 59
+    },
+    "sage.doctest.forker": {
+        "failed": true,
+        "ntests": 432
+    },
+    "sage.doctest.parsing": {
+        "failed": true,
+        "ntests": 316
+    },
+    "sage.doctest.reporting": {
+        "failed": true,
+        "ntests": 124
+    },
+    "sage.doctest.sources": {
+        "failed": true,
+        "ntests": 376
+    },
+    "sage.doctest.test": {
+        "ntests": 23
+    },
+    "sage.doctest.util": {
+        "failed": true,
+        "ntests": 141
+    },
+    "sage.misc.sage_eval": {
+        "failed": true,
+        "ntests": 28
+    },
+    "sage.misc.sage_input": {
+        "failed": true,
+        "ntests": 721
+    },
+    "sage.repl.attach": {
+        "ntests": 128
+    },
```

Comment:
No "failed: false"?

Or "success: true"?

Reply:
No, the timings2.json files created by the doctester don't have these.